pkg/flink/lsp/lsp_completer_ws.go

```diff
@@ -71,7 +71,7 @@ func (w *WebsocketLSPClient) refreshWebsocketConnection() {
 		}
 
 		// we only update client and conn if there was no error, otherwise we leave them as is
-		if lspClient, conn, err := newLSPConnection(w.baseUrl, w.getAuthToken(), w.organizationId, w.environmentId); err == nil {
+		if lspClient, conn, err := newLSPConnection(w.baseUrl, w.getAuthToken(), w.organizationId, w.environmentId, nil); err == nil { //todo - replace nil at the end
```

Comment:
hmm I think this is a bit tricky, I think we need to pass the channel as well, otherwise it won't work after reconnecting right?

Reply:
I think you might need to store the channel in the client when creating it, so that you can reuse it here

Reply:
ohh, good catch - I had added this only to test yesterday

Reply:
done ✅